docs/services/vision/segmentation.md

```diff
@@ -0,0 +1,241 @@
+
+## Configuration
+
+### Add the service and segmenter
+
+Navigate to the [robot page on the Viam app](https://app.viam.com/robots).
+Click on the robot you wish to add the Vision Service to.
+Select the **CONFIG** tab, and click on **SERVICES**.
+
+Scroll to the **Create Service** section.
+To create a [Vision Service](/services/vision/):
+
+1. Select `Vision` as the **Type**.
+2. Enter a name as the **Name**.
+3. Click **Create Service**.
+
+<img src="../../../tutorials/img/try-viam-color-detection/create-service.png" alt="The Create Service panel lists the type as vision and name as vision, with a Create Service button.">
+
+In your Vision Service's panel, add a segmenter into the **Attributes** field.
+For example:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+ "register_models": [
+    {
+        "name": "my_segmenter",
+        "type": "radius_clustering_segmenter",
+        "parameters": {
+            "min_points_in_plane": 2,
+            "min_points_in_segment": 2,
+            "clustering_radius_mm": 3.0,
+            "mean_k_filtering": 0
+        }
+    }
+ ]
+}
+```
+
+Click **SAVE CONFIG** and head to the **COMPONENTS** tab.
+
+{{%expand "You can also configure the entire Vision Service and segmenter in raw JSON" %}}
+
+To add a vision model to your robot, add the `name`, `type`, and `parameters` of the desired segmenter to the `register_models` in the attributes field of the Vision Service config.
+For example:
+
+``` json {class="line-numbers linkable-line-numbers"}
+"services": [
+    {
+        "name": "vision1",
+        "type": "vision",
+        "attributes": {
+          "register_models": [
+            {
+              "name": "my_color_detector",
+              "type": "color_detector",
+              "parameters": {
+                "detect_color" : "#A3E2FF",
+                "hue_tolerance_pct": 0.06,
+                "segment_size_px": 100
+              }
+            },
+            {
+              "name": "my_classifier",
+              "type": "tflite_classifier",
+              "parameters": {
+                "model_path" : "/path/to/model.tflite",
+                "label_path": "/path/to/labels.txt",
+                "num_threads": 1
+              }
+            }
+          ]
+        }
+    }
+]
+```
+
+{{% /expand%}}
+
+### Add a camera component and a "transform" model
+
+You cannot interact directly with the [Vision Service](/services/vision/).
+To be able to interact with the Vision Service you must:
+
+1. Configure a physical [camera component](../../../components/camera) to wrap the service.
+2. Configure a [transform camera](../../../components/camera/transform) to view output from the segmenter overlaid on images from the physical camera.
+
+After adding the component and its attributes, click **SAVE CONFIG**.
+
+Wait for the robot to reload, and then go to the **CONTROL** tab to test the stream of detections.
```

Comment:
Unfortunately we don't have a transform camera for segments yet, so we should remove this section in segmentation